dora/core/client/fs/src/main/java/alluxio/client/file/FileSystemContext.java

```diff
@@ -439,6 +442,11 @@ private synchronized void initContext(ClientContext ctx,
     mBlockMasterClientPool = new BlockMasterClientPool(mMasterClientContext);
     mBlockWorkerClientPoolMap = new ConcurrentHashMap<>();
     mUriValidationEnabled = ctx.getUriValidationEnabled();
+    try {
+      mMembershipManager = MembershipManager.Factory.create(getClusterConf());
+    } catch (IOException ex) {
+      LOG.error("Error setting membership manager.");
```

Comment:
see logging convention https://docs.alluxio.io/os/user/stable/en/contributor/Code-Conventions.html#error-log-level
```suggestion
      LOG.error("Failed to set membership manager.", e);
```

Reply:
done